AppInspector/rules/default/frameworks/rust.json

```diff
@@ -0,0 +1,76 @@
+﻿[
+  {
+    "name": "Development Framework: Rocket",
+    "id": "AI025610",
+    "description": "Development Framework: Rocket",
+    "applies_to": [ "rust" ],
+    "tags": [ "Framework.Development.Rocket" ],
+    "severity": "moderate",
+    "patterns": [
+      {
+        "pattern": "extern crate rocket;|use rocket",
```

Comment:
why does one of these have a semi colon?

Reply:
No reason for the diff other than copy paste.  A semicolon would make it more accurate if there was a chance we would see one without but no diff other than extra character to have to match.  I'll remove.